Makefile

```diff
@@ -59,8 +61,16 @@ formatter:
 	poetry run black rasa tests
 
 lint:
-	poetry run flake8 rasa tests
+     # Ignore docstring errors when running on the entire project
+	poetry run flake8 rasa tests --extend-ignore D
 	poetry run black --check rasa tests
+	make lint-docstrings
+
+branch ?= master # Compare against `master` if no branch was provided
```

Comment:
minor: should we make the variable name uppercase like JOBS above?

Reply:
done